distribution.md

```diff
@@ -24,7 +24,7 @@ A buildpackage MUST exist as either an OCI image on an image registry, an OCI im
 
 A `.cnb` file MUST be an uncompressed tar archive containing an OCI image. Its file name SHOULD end in `.cnb`.
 
-Each FS layer blob in the buildpackage MUST contain a single buildpack at the following file path:
+Each FS layer blob in the buildpackage intended to be a buildpack MUST contain a single buildpack at the following file path:
```

Comment:
I don't understand this change. Don't we dictate that every layer in a buildpackage MUST be a buildpack. When is there a layer in a buildpackage that is not intended to be a buildpack?

Reply:
@ekcasey This is described in detail in the PR description, but the short version is: Windows buildpackages have OS layers that are not buildpacks.

Reply:
@ameyer-pivotal thanks for the clarification, I knew runnable windows images needed some sort of base layer, but I was unaware that even `docker pull` required one!

Reply:
I worry that the wording may be confusing to readers without the explicit windows context. Do we want to say something like "Each FS layer blob must contain a single buildpack with the following exception.. <explain here that a single layer base layer is allow on windows>. Each buildpack layer MUST...".

Reply:
@ameyer-pivotal can we put this context in your git commit as well? This clarification has been super insightful.

Reply:
After chatting with @ekcasey @nebhale and @hone, we'd suggest adding an exception for Windows, as opposed to opening up the buildpackage format to arbitrary layers.

Reply:
@sclevine OK, sounds good. I'll update the wording.

Reply:
@sclevine Actually, something else to consider... The reason for the original approach is that Windows buildpackages may not always look this way. We didn't want to cause a change to the spec if we're ever able to have a scratch equivalent for Windows images in the future (currently it works when packages are remote-only -- it's just pulling with docker that causes the issue). I still think that the spec should comment only on the parts it cares about. How would additional layers impact the platform's ability to look for the layers it's looking for? I.e. what do we gain by being restrictive?

Reply:
@ekcasey @sclevine @jkutner @hone @nebhale Made an update to the wording to make specific callouts for Linux vs Windows packages. After giving it some thought, I'm on board with avoiding opening up buildpackages to arbitrary layers. If Windows buildpackages no longer need OS layers in the future, we can address it then with an RFC if desired.